src/Sulu/Component/Rest/RequestParametersTrait.php

```diff
@@ -25,14 +25,18 @@ trait RequestParametersTrait
      *
      * @param string $name
      * @param bool $force TRUE if value is mandatory
-     * @param mixed $default value if parameter not exists
+     * @param mixed $default value if parameter not exists, this will be ignored if $force is set to true
      *
      * @return string
      *
      * @throws MissingParameterException parameter is mandatory but does not exists
      */
     protected function getRequestParameter(Request $request, $name, $force = false, $default = null)
     {
+        if ($force) {
+            $default = null;
+        }
```

Comment:
would not add this, current state I see it more that it could crash existing applications if we change this one.

Reply:
Removed it.